### ISSUE TITLE
ML-589 ⁃ ♿ Images need alt text

### ISSUE THREAD

**Issue by binarybottle** [ChildMindInstitute/mindlogger-app#596]
> The required alt attribute specifies an alternate text for an image, if the image cannot be displayed.

― [W3C Schools HTML <img> alt Attribute](https://www.w3schools.com/tags/att_img_alt.asp)

> 6. Web Content Accessibility Guidelines
> Guideline 1. Provide equivalent alternatives to auditory and visual content.

― [W3C Web Content Accessibility Guidelines 1.0](https://www.w3.org/TR/WCAG10/#gl-provide-equivalents)

> Browsers do not always display images. For example: 
> * Non-visual browsers (such as those used by people with visual impairments)
> * The user chooses not to display images (saving bandwidth, privacy reasons)
> * The image is invalid or an [unsupported type](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Supported_image_formats)
> 
> In these cases, the browser may replace the image with the text in the element's `alt` attribute. For these reasons and others, provide a useful value for `alt` whenever possible.

― [<img>: The Image Embed element - HTML: Hypertext Markup Language | MDN|https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-alt]

> Principle 1 – Perceivable
> Information and user interface components must be presentable to users in ways they can perceive.
> 
> Guideline 1.1 – Text Alternatives
> Provide text alternatives for any non-text content so that it can be changed into other forms people need, such as large print, braille, speech, symbols or simpler language.

― [How to Meet WCAG (Quickref Reference)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#text-alternatives)

[:link: Web Accessibility Tutorials: An alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/)
